package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -1269,3 +1269,27 @@ void hostapd_ubus_notify(struct hostapd_data *hapd, const char *type, const u8 *
 
 	ubus_notify(ctx, &hapd->ubus.obj, type, b.head, -1);
 }
+
+void hostapd_ubus_notify_beacon_report(struct hostapd_data *hapd, const u8 *addr, u8 token, u8 rep_mode, struct rrm_measurement_beacon_report *rep, size_t len)
```

Comment:
WARNING: line over 80 characters
#53: FILE: package/network/services/hostapd/src/src/ap/ubus.c:1275:
+void hostapd_ubus_notify_beacon_report(struct hostapd_data *hapd, const u8 *addr, u8 token, u8 rep_mode, struct rrm_measurement_beacon_report *rep, size_t len)

Reply:
I tried to use the coding style of the file. But I'm not sure if this is okay. :S